README.md

```diff
@@ -4,3 +4,35 @@ This repository contains ports of Tock OS (https://www.tockos.org) to Titan
 chips.
 
 This is not an officially supported Google product.
+
+
+## Getting started
+
+### Clone the repo
+
+Get the source:
+
+```shell
+git clone --recursive git@github.com:google/tock-on-titan.git
```

Comment:
I'd use the HTTP URL here (`https://github.com/google/tock-on-titan.git`) instead of SSH.

Reply:
Done.